frame/composable-traits/src/vault.rs

```diff
@@ -112,6 +112,11 @@ pub trait Vault {
 		vault_id: &Self::VaultId,
 		lp_amount: Self::Balance,
 	) -> Result<Self::Balance, DispatchError>;
+
+	fn calculate_lp_tokens_from_asset_amount(
```

Comment:
in AMM DEX traits (pablo) they call it `amount_of_lp_token_for_added_liquidity`

Reply:
I have renamed the function in the last commit together with the refactor you suggested above